packages/core/src/Address/util.ts

```diff
@@ -15,8 +15,8 @@ export const isAddressWithin =
     addresses.includes(address!);
 
 /**
- * Receives a transaction and a set of addresses to check if the transaction is outgoing,
- * i.e., some of the addresses are included in the transaction inputs
+ * Receives a transaction and a set of addresses to check if
+ * some of them are included in the transaction inputs
  */
-export const isOutgoing = (tx: TxAlonzo, ownAddresses: Address[]): boolean =>
+export const isSending = (tx: TxAlonzo, ownAddresses: Address[]): boolean =>
```

Comment:
Your rename makes it better, but I'm thinking we should move it under inspectors as it's inspecting a tx object. Also, how about this? Even more explicit:

```suggestion
export const anyInputFromAddress = (tx: TxAlonzo, ownAddresses: Address[]): boolean =>
```